apps/demos/Demos/Scheduler/Templates/Vue/AppointmentTemplate.vue

```diff
@@ -35,20 +43,6 @@ const movieData = getMovieById(props.templateModel.targetedAppointmentData?.movi
 
 </script>
 <style scoped>
-  .dx-tooltip-wrapper .dx-overlay-content .dx-popup-content {
-    padding: 14px;
-  }
-
-  .showtime-preview > div:first-child {
-    font-size: 12px;
-    white-space: normal;
-  }
-
-  .showtime-preview > div:not(:first-child) {
-    font-size: 11px;
-    white-space: normal;
-  }
-
   .movie-tooltip .movie-info {
```

Comment:
Let's remove these styles too. Because of them the appt tooltip is looking incorrectly